HW_04/Task_5/Program.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Task_5
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            char[] alphabet = new char[26];
+            int letter = 65;
+
+            for (int i = 0; i < alphabet.Length; i++)
+            {
+                alphabet[i] = (char)letter;
+
+                if (i < alphabet.Length -1)
```

Comment:
you do not need it if you write like below:

`for (int i = 0; i < alphabet.Length -1 ; i++)`